src/main/java/com/example/demo/feed/domain/Feed.java

```diff
@@ -21,32 +21,31 @@
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
 
-
-
     @Embedded
     private Writer writer;
 
     private String content;
 
     private Long likeNum;
 
+    private LocalDateTime wrtTime;
+
+    @Column(name = "is_delete")
```

Comment:
```suggestion
    @Column(name = "deleted")
```
칼럼명에는 is라는 내용을 넣지마세요